test/libsolidity/semanticTests/viaYul/array_storage_index_zeroed_test.sol

```diff
@@ -1,48 +1,46 @@
 contract C {
-	uint[] storageArray;
-	function test_zeroed_indicies(uint256 len) public
-	{
-		storageArray.length = len;
+    uint[] storageArray;
+    function test_zeroed_indicies(uint256 len) public
+    {
+        storageArray = new uint[](len);
 
-		for (uint i = 0; i < len; i++)
-			storageArray[i] = i + 1;
+        for (uint i = 0; i < len; i++)
+            storageArray[i] = i + 1;
 
-		if (len > 3)
-		{
-			storageArray.length = 3;
+        if (len > 3)
+        {
+            storageArray = new uint[](3);
 
-			for (uint i = 3; i < len; i++)
-			{
-				assembly {
-					mstore(0, storageArray_slot)
-					let pos := add(keccak256(0, 0x20), i)
+            for (uint i = 3; i < len; i++)
+            {
+                assembly {
+                    mstore(0, storageArray_slot)
+                    let pos := add(keccak256(0, 0x20), i)
 
-					if iszero(eq(sload(pos), 0)) {
-						revert(0, 0)
-					}
-				}
-			}
+                    if iszero(eq(sload(pos), 0)) {
+                        revert(0, 0)
+                    }
+                }
+            }
 
-		}
+        }
 
-		storageArray.length = 0;
-		storageArray.length = len;
+        storageArray = new uint[](0);
+        storageArray = new uint[](len);
 
-		for (uint i = 0; i < len; i++)
-		{
-			require(storageArray[i] == 0);
+        for (uint i = 0; i < len; i++)
+        {
+            require(storageArray[i] == 0);
 
-			uint256 val = storageArray[i];
-			uint256 check;
+            uint256 val = storageArray[i];
+            uint256 check;
 
-			assembly { check := iszero(val) }
+            assembly { check := iszero(val) }
 
-			require(check == 1);
-		}
-	}
+            require(check == 1);
+        }
+    }
 }
-// ====
-// compileViaYul: true
```

Comment:
Would it hurt to implement ``push`` for Yul codegen here as well? It looks like using ``m_utils.resizeDynamicArrayFunction`` in the ``MemberAccess`` visiter there or something like that should just work.